### PR TITLE
converted characters from string format to character format

### PR DESCRIPTION
**Pull request by sameergupta4873**
### Description
Converted the characters written in string format to character format in moveit2 per #871 Char is char, String is string

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/ros-planning/moveit/blob/master/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://ros-planning.github.io/moveit_tutorials/doc/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit_experimental/kinematics_cache/v2/kinematics_cache/src/kinematics_cache.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*********************************************************************
+' '/*********************************************************************
```
Comment (henningkayser) [inline moveit_experimental/kinematics_cache/v2/kinematics_cache/src/kinematics_cache.cpp:1]: please revert

**Review by henningkayser — COMMENTED** [review]
Thank you! Changes look good, I just tagged a misplaced quote that should be removed before merging.

**Comment by sameergupta4873** [comment]
Welcome @henningkayser, changes been made as suggested

**Review by henningkayser — APPROVED** [review]
Thank you!

**Review by vatanaksoytezer — APPROVED** [review]

**Pull request merged**